tests/python/pytorch/graphbolt/impl/test_csc_sampling_graph.py

```diff
@@ -1218,6 +1218,22 @@ def test_from_dglgraph_homogeneous():
     dgl_g = dgl.rand_graph(1000, 10 * 1000)
     gb_g = gb.from_dglgraph(dgl_g, is_homogeneous=True)
 
+    # Get the COO representation of the CSCSamplingGraph.
+    gb_coo = [
```

Comment:
Another choice to get column is
```
num = indptr[1:] - indptr[:-1]
column = torch.arange(indptr.size(0))
column = column.repeat_interleave(num)
```

you can choose the one you like

Reply:
Done